app/views/users/_main_feed.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if @pinned_stories.present? %>
   <div class="pinned-articles">
-    <div class="pinned-articles-indicator"><img src="<%= asset_path("pushpin.svg") %>" alt="pin" /> Pinned</div>
+    <h5><img src="<%= asset_path("pushpin.svg") %>" width="20" height="20" alt="pin" /> Pinned</h5>
```

Comment:
should we rather add a class here instead of inline width, height css?

Reply:
Defining width and height is good convention: if CSS is not (fully) loaded the image can otherwise see huge and cause unnecessary layout jump.

Reply:
cool - sounds good, didn't think of that!